src/pages/components/colorpickers.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: Color picker
+description: >
+  The color picker component allows users to choose a color using a variety of input methods.
+package: '@zendeskgarden/react-colorpickers'
+components:
+  - colorpickers/src/elements/Colorpicker/index.tsx
+  - colorpickers/src/elements/ColorpickerDialog/index.tsx
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>To let users change or update brand colors</li>
+      <li>To let users configure chart colors</li>
+    </ul>
+  </Use>
+</Usage>
+
+## How to use it
+
+### Color picker
```

Comment:
Use `Default` for consistency.

Reply:
good call, i've added some copy and an example to our drafts.